core/src/test/java/org/hibernate/ogm/utils/OgmTestCase.java

```diff
@@ -123,4 +124,14 @@ public void inTransaction(Consumer<Session> consumer) {
 		}
 	}
 
+	protected <T> void deleteAll(Class<T> entity, Serializable... ids) {
```

Comment:
Can you just call this `delete`? I prefer to keep the suffix `All` only for cases where it's not possible to specify the ids.

Reply:
delete was my first choice too. Unfortunately many implementations of *OGMBaseTest* already use that name :(

Reply:
WDYT about deleteEntities?

Reply:
No need to change it, leave it `deleteAll`

Reply:
ok